lib/ansible/runner/__init__.py

```diff
@@ -590,8 +590,10 @@ def _executor(self, host, new_stdin):
             msg = str(ae)
             self.callbacks.on_unreachable(host, msg)
             return ReturnData(host=host, comm_ok=False, result=dict(failed=True, msg=msg))
-        except Exception:
+        except Exception, e:
             msg = traceback.format_exc()
+            if hasattr(e, "_ansible_srcinfo"):
+                msg = ("%(varname)s from %(basedir)s\n" % e._ansible_srcinfo) + msg
```

Comment:
Ideally, there would be more information available, but this will at least let you know what is failing to evaluate, instead of just that... something somewhere is invalid.